api-2.2/src/test/java/org/openmrs/module/fhir/api/ConditionServiceTest.java

```diff
@@ -102,4 +111,25 @@ public void createFHIRCondition_shouldReturnCreatedFHIRCondition() {
 		assertEquals(justCreatedCondition.getClinicalStatus(), getSavedCondition.getClinicalStatus());
 		assertEquals(justCreatedCondition.getOnset(), getSavedCondition.getOnset());
 	}
+
+	@Test
+	public void getConditionsByPatientUuid_shouldReturnListOfActiveConditions() {
+		Condition condition = new Condition();
+		IdType id = new IdType();
+		id.setValue(CONDITION_UUID);
+		condition.setId(id);
+		Patient patientRef = Context.getPatientService().getPatient(2);
+		Reference patient = FHIRUtils.buildPatientOrPersonResourceReference(patientRef);
+		condition.setSubject(patient);
+		condition.setClinicalStatus(Condition.ConditionClinicalStatus.ACTIVE);
+		condition.setCode(new CodeableConcept().addCoding(
+				new Coding().setCode("999").setDisplay("NO").setSystem(FHIRConstants.OPENMRS_URI)).setText("NO"));
+
+		getService().createFHIRCondition(condition);
+		List<Condition> activeConditions = getService().getConditionsByPatientUuid(patientRef.getUuid());
```

Comment:
Can you assert that the list was empty before saving? Or at least less than after saving?